lib/src/index.ts

```diff
@@ -324,6 +324,9 @@ export { default as NxSystemNotice } from './components/NxSystemNotice/NxSystemN
 
 export { default as NxProgressBar } from './components/NxProgressBar/NxProgressBar';
 
+export { default as NxToastContainer } from './components/NxToast/NxToastContainer';
+export { default as NxToast } from './components/NxToast/NxToast';
```

Comment:
Don't forget to export the props

Reply:
> Don't forget to export the props

Updated exports here: [5277f2f](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/5277f2f440a29e543bf1cbde2fa2614027b96643)